go/vt/vttablet/tabletmanager/vreplication/framework_test.go

```diff
@@ -503,7 +512,7 @@ func expectNontxQueries(t *testing.T, queries []string) {
 		select {
 		case got = <-globalDBQueries:
 
-			if got == "begin" || got == "commit" || got == "rollback" || strings.Contains(got, "update _vt.vreplication set pos") {
+			if got == "begin" || got == "commit" || got == "rollback" || strings.Contains(got, "update _vt.vreplication set pos") || heartbeatRe.MatchString(got) {
```

Comment:
Is there any risk at all for some query to _contain_ the text `update _vt.vreplication set pos`? Does it make sense to convert `strings.Contains()` to `strings.HasPrefix()`?

Reply:
That's a good question. I'm not sure why the original author of this test helper used Contains. It might be in an effort to strip potential whitespace? Or potentially to strip leading `/` which somehow end up in some of the test queries at the start?